index.bs

```diff
@@ -3479,7 +3487,9 @@ optionally evidence of [=user consent=] to a specific transaction.
 
         [=[RPS]=] that wish to support a wide range of [=authenticators=] SHOULD include at least the following {{COSEAlgorithmIdentifier}} values:
 
-        * -8 (Ed25519)
+        * -50 (Ed25519)
+        * -8 (EdDSA)
+        * -9 (ESP256)
```

Comment:
IANA is showing that it is unassigned as of now. -7 is for ECDSA with P-256 with SHA256.

Reply:
-9 and -19 are registered now: https://www.iana.org/assignments/cose/cose.xhtml#algorithms